modules/complete-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="complete-assisted-installer-oci_{context}"]
+= Completing the remaining Assisted Installer steps
+
+After you provision {oci-first} resources and upload {product-title} custom manifest configuration files to {oci}, you must complete the remaining cluster installation steps on the Assisted Installer before you can create an instance {oci}.
+
+.Prerequisites
+
+* You created a resource stack on {oci}, and the stack includes the custom manifest configuration files and {oci} Resource Manager configuration resources.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console] web console, go to the **Host discovery** step. Under the **Role** column, assign a node role, `Control plane node` or `Worker`, for each targeted hostname.
```

Comment:
One more clarification that we should wait until all nodes appears in Host discovery and have status "Ready". 
Otherwise we cannot continue the installation.

Reply:
HI @yobshans . Added:

![Screenshot from 2024-02-15 12-39-01](https://github.com/openshift/openshift-docs/assets/57954076/b4767eaa-ead3-4234-a41c-8e0dd0dc0883)